esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -2595,15 +2623,38 @@ void expandQueryActionTargetList(IProperties *queryIds, IPropertyTree *queryset,
                     const char *name = active->query().queryProp("@name");
                     const char *id = active->query().queryProp("@id");
                     if (name && id && WildMatch(name, itemId))
-                        queryIds->setProp(id, 0);
+                    {
+                        if (suspendedByUser)
```

Comment:
If both --suspended-by-user and --activated=yes or no is selected, should they both have to be true?

"Unpublish queries that are active=no and suspended"?

Reply:
The relation has been changed to AND.